cmd/node/main.go

```diff
@@ -878,6 +889,9 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		return err
 	}
 
+	chanLogRewrite <- true
```

Comment:
the application will lock here if the node is not started in termui mode because no one will read on those channels. Make them buffered?

Reply:
thanks. done